src/AppInstallerCLITests/MsixInfo.cpp

```diff
@@ -64,3 +55,37 @@ TEST_CASE("MsixInfo_WriteFile", "[msixinfo]")
 
     REQUIRE(1 == std::filesystem::file_size(file));
 }
+
+TEST_CASE("MsixInfo_ValidateMsixTrustInfo", "[msixinfo]")
+{
+    if (!Runtime::IsRunningAsAdmin())
+    {
+        INFO("Test requires admin privilege. Skipped.");

```

Comment:
```suggestion
        WARN("Test requires admin privilege. Skipped.");
```
Not sure how this behaves instead, maybe it is more obvious?